lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -964,16 +964,16 @@ def view_init(self, elev=None, azim=None):
         """
 
         self.dist = 10
-
+        # normalize the angle to stay between [-180, 180]
         if elev is None:
-            self.elev = self.initial_elev
+            self.elev = art3d.norm_angle(self.initial_elev)
         else:
-            self.elev = elev
+            self.elev = art3d.norm_angle(elev)
```

Comment:
If you're going to normalize it at all, elevation should be normalized to `[-90, +90]`

Reply:
looking at code here: https://github.com/jinshifen33/matplotlib/blob/master/lib/mpl_toolkits/mplot3d/axes3d.py#L1214  , the angle is kept in [-180, 180]